consensus-genome/test/test_wdl.py

```diff
@@ -142,31 +153,33 @@ def test_sars_cov2_ont_cg_input_file_format(self):
                 self.assertGreater(os.path.getsize(filename), 0)
 
     def test_zip_outputs(self):
-        res = self.run_miniwdl(task="ZipOutputs", args=["prefix=test", f"outputFiles={self.wdl}"])
+        res = self.run_miniwdl(task="ZipOutputs", args=[
+            "prefix=test",
+            f"outputFiles={self.wdl}",
+        ])
         with zipfile.ZipFile(res["outputs"]["ZipOutputs.output_zip"]) as fh:
             self.assertEqual(fh.namelist(), ["run.wdl"])
 
     def test_fetch_sequence_by_accession_id(self):
-        res = self.run_miniwdl(task="FetchSequenceByAccessionId", args=["accession_id=NC_000913.3"])
-        with open(res["outputs"]["FetchSequenceByAccessionId.sequence_fa"]) as fh:
-            self.assertEqual(fh.readline().strip(), ">NC_000913.3")
-            self.assertTrue(fh.readline().startswith("AGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTG"))
-
-        with self.assertRaises(CalledProcessError) as ecm:
-            self.run_miniwdl(task="FetchSequenceByAccessionId", args=["accession_id=NO_ACCESSION_ID"])
-        self.assertRunFailed(ecm, task="FetchSequenceByAccessionId",
-                             error="AccessionIdNotFound",
-                             cause=("The Accession ID was not found in the CZ ID database, "
-                                    "so a generalized consensus genome could not be run"))
-
-    def test_fetch_sequence_by_expired_accession_id(self):
```

Comment:
I removed this test because NR/NT should have expired versions so the increment logic shouldn't be needed anymore.